index.html

```diff
@@ -4,44 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>mobile</title>
+  <title>About Me</title>
   <link rel="stylesheet" type="text/css" href="style.css"/>
 </head>
 <body>
 <!-- This nav is for Hour,signal,Wifi and Baterry percentaje -->
   <header>
-    <nav>
+    <nav class="background-image">
       <ul class="nav-container">
-        <li class="nav-items"><a href="#"></a>MyLogo</li>
-        <li class="nav-items assetNav"><a href="#"></a></li>
+        <a href="#" class="nav-items">Facundo Malgarini</a>
+        <a href="#" class="nav-items assetNav"></a>
+        <a href="#" class="nav-items nav-links navPortfolio">Porfolio</a>
+        <a href="#" class="nav-items nav-links navAbout">About</a>
+        <a href="#" class="nav-items nav-links navContact">Contact</a>
```

Comment:
Hello @fmalgarini, you did a great job with your portfolio, well-done 👍 , kindly adjust your navigation items such that it is aligned to the right while the `logo` is aligned to the left just as it is in the figma design given 😄
Also, you should reduce the size of your background images and place them in the right position away from the text as seen in the template given 👍  

Figma design :
![image](https://github.com/fmalgarini/My-portfolio-Project/assets/20537712/86efb604-127a-4ba6-b09c-d904f92e17cd)

Your design :
![image](https://github.com/fmalgarini/My-portfolio-Project/assets/20537712/a0e87283-dae9-4072-a213-d5698cf9a83a)